packages/babel-plugin-extract-messages/src/index.ts

```diff
@@ -27,15 +30,15 @@ function addMessage(
       throw path.buildCodeFrameError(
         "Different defaults for the same message ID."
       )
-    } else {
```

Comment:
small refactoring, reduced indentation level since `throw` will exit the function anyway

Reply:
👍